taxi_service/settings.py

```diff
@@ -124,6 +126,11 @@
 
 STATIC_URL = "static/"
 
+STATICFILES_DIRS = [
+    # BASE_DIR / "static",
+    os.path.join(BASE_DIR, 'static'),
+]
```

Comment:
The STATICFILES_DIRS setting includes a commented-out line for BASE_DIR / 'static'. Ensure that the correct path is uncommented and used according to your project's structure and requirements.